civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6689,7 +6691,12 @@ function _civihr_employee_portal_create_default_reports_configuration() {
 function civihr_employee_portal_get_markup_for_extension($resource = '', $scriptExtensionKey, $markupModuleName) {
   $script = CRM_Core_Config::singleton()->debug ? "{$resource}.js" : "{$resource}.min.js";
 
-  // Adds RequireJS + AngularJS dependencies only once
+  // Adds RequireJS library only once
```

Comment:
i think you can write it like this, so it's more compact
```js
// Adds RequireJS + AngularJS dependencies only once
if (!strpos(drupal_get_js('footer'), 'require.min.js')) {
  civicrm_resources_load('org.civicrm.reqangular', ['require.min.js']);
  civicrm_resources_load('org.civicrm.reqangular', ['reqangular.min.js']);
}
```
i think it's safe to assume that if `require.min.js` is not present, neither it's reqangular?